android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -766,7 +767,9 @@ private boolean requestAudioFocus() {
     }
 
     private void setPlayWhenReady(boolean playWhenReady) {
-        if (player == null) {
+        PowerManager powerManager = (PowerManager) getContext().getSystemService(Context.POWER_SERVICE);
```

Comment:
can we not create an instance of the power manager at a higher level ?

Reply:
Done 👍